objects/Fan-MadeScenariosCampaignsMiscellany.66e97c/Fan-MadeCampaigns.89c32e.json

```diff
@@ -43,7 +43,7 @@
     "CloseEncountersoftheLV-426Kind.4f5421",
     "CelticRising.4d305a",
     "CallofthePlaguebearer.613b64",
-    "BloodborneCityoftheUnseen.24fb2b",
+    "Bloodborne-CityoftheUnseen.24fb2b",
```

Comment:
This should be removed from this PR since it stop the mod from bundling. Known issue that is resolved with https://github.com/argonui/SCED/pull/298